sycl/doc/extensions/PlatformContext/PlatformContext.adoc

```diff
@@ -0,0 +1,17 @@
+= SYCL(TM) Proposals: Platform Default Contexts
+James Brodman <james.brodman@intel.com>
+v0.1
+:source-highlighter: pygments
+:icons: font
+:y: icon:check[role="green"]
+:n: icon:times[role="red"]
+
+== Platform Default Contexts
+
+This extension adds the notion of a default SYCL context per SYCL platform. The default context for each platform contains all devices in the platform.
+
+The platform class gains three new methods: `get_default_context`, `push_default_context`, and `pop_default_context`.
```

Comment:
It would be nice if functions are described in a table similar to how it's done in other extensions, like https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/ParallelForSimplification/SYCL_INTEL_parallel_for_simplification.asciidoc. Otherwise, at least, their signature is not clear.

I'm OK if it is done in a separate PR.